knockout_modeler/tests.py

```diff
@@ -50,25 +54,23 @@ class Person(f.FakeModel):
     home_city = models.ForeignKey(HomeCity, blank=True)
 
     class Meta:
-        app_label = 'test'  
-
-    def knockout_fields(self):
-        return [
-                    'first_name', 
-                    'home_city',
-                    'profession'
-                ]
+        app_label = 'test'
 
     @classmethod
-    def comparator(self):
+    def comparator(cls):
         return 'uuid'
 
+    @staticmethod
+    def knockout_fields():
+        return ['first_name', 'home_city', 'profession']
+
 ####
 # The actual tests
 ####
 
-class KnockoutTests(TestCase):
-    
+
+class KnockoutTests(TransactionTestCase):
```

Comment:
`TransactionTestCase` allowed `setup_user` to be called multiple time in a single test case.